### PR TITLE
Log component

### PR DESCRIPTION
**Pull request by ejewik**

### CODE REVIEW

**Comment by ejewik** [comment]
@rahuldadlani03 Hello! I created this PR to merge into landing-page since the development branch doesn't have work in the source directory yet, but let me know if I should close this and create another PR to merge into development instead. I added the fixes you mentioned over DM (Thank you for the feedback!!) and added a few media queries to make the component responsive. Let me know if this looks alright or if I should revise anything else.

File: source/components/monthly-log-component.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+class logComponent extends HTMLElement {
```
Comment (rahuldadlani03) [inline source/components/monthly-log-component.js:1]: Replace logComponent with MonthlyLogComponent

File: source/components/monthly-log-component.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+        if (month) {
+            this.shadowRoot.querySelector('.log-calendar-month-text').innerText = month;
+        } else {
+            this.shadowRoot.querySelector('.log-calendar-month-text').innerText = "JAN";
+        }
+    }
+}
+
+customElements.define('log-component', logComponent);
```
Comment (rahuldadlani03) [inline source/components/monthly-log-component.js:167]: - Replace `log-component` with `monthly-log-component`
- Replace logComponent with MonthlyLogComponent

**Review by rahuldadlani03 — COMMENTED** [review]
This wasn't mentioned this in a style guide anywhere but just to keep the naming scheme of components, it would be helpful if you renamed the class and the name of the component so it's in line with the name of the file

File: source/components/monthly-log-component.js (hunk 2 above)
Comment (ejewik) [reply source/components/monthly-log-component.js]: @rahuldadlani03 Hi! I added the naming edits you mentioned in the comments and also applied them to the CSS classes as well, just appending 'monthly' to each CSS class. My bad, renaming the class itself slipped my mind. I requested review again. Thanks!

**Pull request merged**